io.github.JakubMelka.Pdf4qt.yml

```diff
@@ -0,0 +1,28 @@
+app-id: io.github.JakubMelka.Pdf4qt
+runtime: org.kde.Runtime
+runtime-version: 5.15-22.08
+sdk: org.freedesktop.Sdk
```

Comment:
the SDK doesn't match the runtime.

Reply:
I didn't know if I could use both the kde runtime and the freedesktop runtime together.

Reply:
the KDE runtime "includes" the freedestkop runtime. It's a superset.